src/components/DateInput/DateInput.tsx

```diff
@@ -12,7 +13,15 @@ type DateInputProps = {
 
 export const DateInput = forwardRef(
   (
-    { className, describedBy, id, isInvalid, isRequired, name }: DateInputProps,
+    {
+      className,
+      describedBy,
+      disabled,
```

Comment:
Could we use `isDisabled`? I think that is more consistent with the prop names of the other booleans.